pengine/allocate.c

```diff
@@ -2145,6 +2146,11 @@ stage8(pe_working_set_t * data_set)
 
     crm_log_xml_trace(data_set->graph, "created resource-driven action list");
 
+    /* adding pseudo action that carries list of nodes in maintenance */
+    crm_trace("adding pseudo action holding nodes in maintenance");
+    action = get_pseudo_op(CRM_OP_MAINTENANCE_NODES, data_set);
+    set_bit(action->flags, pe_action_print_always);
+
```

Comment:
reminder for myself - current implementation in remote_ra_process_pseudo probably doesn't work like that for multiple nodes in maintenance ...